Shared/AuthenticationKeychainInfo.swift

```diff
@@ -33,7 +33,7 @@ open class AuthenticationKeychainInfo: NSObject, NSCoding {
     open var useTouchID: Bool
 
     // Timeout period before user can retry entering passcodes
-    open var lockTimeInterval: TimeInterval = 15 * 60
+    private let lockTimeInterval: TimeInterval = AppConstants.BuildChannel == .release ? 15 * 60 : 60
```

Comment:
I decreased lockout timer for non-release builds to make it easier to test

Reply:
- [ ] Please use `.isRelease` variable
- [ ] Use the newer API `15.minutes` and `1.minutes/60.seconds`

Reply:
Can't use `minutes` here, this is `Shared` module and `BraveShared` can't be imported here (`Circular dependency between modules 'Shared' and 'BraveShared'`)